src/libs/history.c

```diff
@@ -882,7 +886,7 @@ static gboolean _changes_tooltip_callback(GtkWidget *widget, gint x, gint y, gbo
     add_blend_history_change_enum(blend_cst, _("colorspace"), dt_develop_blend_colorspace_names);
     add_blend_history_change_enum(mask_mode, _("mask mode"), dt_develop_mask_mode_names);
     add_blend_history_change_enum(blend_mode, _("blend mode"), dt_develop_blend_mode_names);
-    add_blend_history_change(blend_parameter, "%.2f EV", _("blend fulcrum"));
+    add_blend_history_change(blend_parameter, _("%.2f EV"), _("blend fulcrum"));
```

Comment:
Yes, nice catch before the string freeze!

Reply:
It seems it was already translated anyway, so it wont have broken the translation but better have it translated anyway.